src/AccessibilityInsights.SharedUx/Controls/CustomControls/ToggleMenuItemAutomationPeer.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using AccessibilityInsights.SharedUx.Utilities;
+using System.Windows.Automation;
+using System.Windows.Automation.Peers;
+using System.Windows.Automation.Provider;
+using System.Windows.Controls;
+
+namespace AccessibilityInsights.SharedUx.Controls.CustomControls
+{
+    internal class ToggleMenuItemAutomationPeer : MenuItemAutomationPeer, IToggleProvider
+    {
+        private readonly RadioButton _radioButton;
+
+        public ToggleMenuItemAutomationPeer(MenuItem item)
+            : base(item)
+        {
+            _radioButton = item.Header as RadioButton;
```

Comment:
Could malformed XAML cause _radioButton to be null? If so, would it make sense to throw an Exception here? ArgumentException wouldn't make sense, so it would need to be some other type